site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/model/SiteConfiguration.java

```diff
@@ -31,9 +31,30 @@ public SiteConfiguration(String name, String displayName, List<SiteAttributeConf
         this.attributes = attributes;
     }
 
+    /**
+     * Site Constructor with two parameters
+     *
+     * @param name the Site name
+     * @param displayName the Site displayName
+     * @param type the Site type
+     * @param attributes the List of SiteAttributeConfiguration attributes
+     * @param child the List of SiteAttributeConfiguration child / children.
+     *
+     */
+    public SiteConfiguration(String name, String displayName, String type,
```

Comment:
There are now 3 constructors for SiteConfiguration, can we just have one? In real use, only the default constructor is used I think (for deserialization) - the other constructors are just used by tests (which could use the setters instead).

Reply:
Removed the unused constructor